Make adjustments to Feature PR templates (#269)
KinsonDigital/Velaptor: .github/PULL_REQUEST_TEMPLATE/preview-feature-pr.md

<!--
    !! NOTE !! - ONLY PROJECT OWNERS AND MAINTAINERS CAN CREATE PRODUCTION AND PREVIEW RELEASE PULL REQUESTS
    Please use the "preview-feature-pr" pull request template if you have contributions to make.
-->
<!--suppress HtmlDeprecatedAttribute -->
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

> - [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch with the correct naming syntax in the following example.

Reviewer: Use colon, not period at the end and use commas in the middle of the sentence.
```suggestion
- [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
```

  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

> - [ ] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch with the correct naming syntax in the following example.

Reviewer: Use colon, not period at the end and use commas in the middle of the sentence.
```suggestion
- [ ] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
```

  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

> - [ ] The **_base(target)_** branch for this pull request is a **_preview release_** branch with the correct naming syntax in the following example.

Reviewer: Use colon, not period at the end and use commas in the middle of the sentence.
```suggestion
- [ ] The ***_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
```

>   - A **_base(target)_** branch is the branch that is a source branch being merged into it.

Author: ❔

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more info on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [ ] Pull request title matches the title of the associated issue.

- [ ] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [ ] The labels attached to this PR match the labels attached to the associated issue.

- [ ] My code follows the coding style of this project.

>   - The style is enforced by the *.editorconfig* files in the project and displayed as warnings.  If there is an edge case with a coding style that should be ignored or changed, reach out, and let's discuss it.

Author: ❔

- [ ] A ![preview-label](https://user-images.githubusercontent.com/85414302/150838564-33f6044b-55f9-4dd9-8783-1d739de9d92f.png) label has been added to the pull request.

- [ ] I have manually tested my code changes to the best of my ability.

- [ ] All tests passed locally.

>   - This is required because unit tests might pass locally but not in the CI environment during the status check process or vise-versa.

Reviewer: Misspelled vice
```suggestion
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
```
